### DISCUSSION TITLE
Vendor Prefixes Support

### DISCUSSION THREAD

**Discussion by sewalsh** [unocss/unocss#2776] [General]
Is there an unocss way of enabling vendor autoprefixes on production builds?

Prefixes are present with `dev`, not `build`.

**Comment by antfu** [comment]
We could probably have an autoprefixer postprocessor package via using this API
https://github.com/unocss/unocss/blob/b07b50d69220928eacfcf052453c2f7ce8245f47/packages/core/src/types.ts#L298-L301

PR welcome

**Comment by zouhangwithsweet** [comment]
Does it can help ? https://github.com/zouhangwithsweet/unocss-preset-autoprefixer
